test/WebSites/BasicWebSite/Controllers/ActionConstraints/ConsumesAttribute_WithFallbackActionController.cs

```diff
@@ -10,20 +10,20 @@ namespace BasicWebSite.Controllers.ActionConstraints
     public class ConsumesAttribute_WithFallbackActionController : Controller
```

Comment:
The action selection component of it, you can have [Consumes] globally to express that your API only supports json for example, or you can use consumes to disambiguate between two actions (action selection again).